volume/local/local_unix.go

```diff
@@ -136,6 +138,22 @@ func (v *localVolume) mount() error {
 			mountOpts = strings.Replace(mountOpts, "addr="+addrValue, "addr="+ipAddr.String(), 1)
 		}
 	case "cifs":
+		resolveDeviceName := true
```

Comment:
I see you pushed; just now thinking (it’s a nit really!); perhaps use `skipResolve` or skipXXXX for the variable? (false by default)

Reply:
Done! 😄